game_for_class.py

```diff
@@ -8,70 +8,46 @@
 # Start with the player's class
 class Character:
     """All the methods the game player can use"""
-    def __init__(self,
-                 name: str,
-                 hp: float,
-                 attack: float,
-                 defense: float,
-                 speed: float,
-                 team: int):
+    def __init__(self, name: str, hp: float, attack: float, defense: float, speed: float, team: int):
```

Comment:
The old formatting with each argument getting it's own line is more readable

Reply:
Issue was that you said, the formatting was messed up

Reply:
My complaint was with the random whitespace line cutting the whole thing up.